packages/components/user/__tests__/user.test.tsx

```diff
@@ -19,10 +20,30 @@ describe("User", () => {
   });
 
   it("should have the passed name", () => {
-    const {container} = render(<User name="Test" />);
-    const name = container.querySelector("span");
+    const {getByText} = render(<User name="Test" />);
+    const nameElement = getByText("Test");
 
-    expect(name).toHaveTextContent("Test");
+    expect(nameElement).toBeInTheDocument();
+  });
+
+  it("should render the default icon if no name passed to avatar", () => {
+    const {queryByTestId} = render(
+      <User
+        avatarProps={{
+          radius: "full",
+          size: "sm",
+          icon: <AvatarIcon />,
+          classNames: {
+            base: "bg-gradient-to-br from-[#FFB457] to-[#FF705B]",
+            icon: "text-black/80",
+          },
+          name: "avatar-icon",
+        }}
+        name="Test"
+      />,
+    );
+
+    expect(queryByTestId("avatar-icon")).toBeNull();
```

Comment:
this test case doesn't match with the title. also the issue is `name` passing to avatar component. in your test case, it shouldn't have `name` in `avatarProps`.

Reply:
Ok, I Will try  again to make it perfect.